horreum-web/src/components/ConditionComponent.tsx

```diff
@@ -8,45 +8,46 @@ type ConditionComponentProps = {
     value: any
     onChange(value: any): void
     isTester: boolean
+    properties: any
 } & ConditionComponentDef
 
-export default function ConditionComponent(props: ConditionComponentProps) {
+export default function ConditionComponent({ value, onChange, isTester, type, properties, title, name, description }: ConditionComponentProps) {
     let component
-    switch (props.type) {
+    switch (type) {
         case "LOG_SLIDER":
             component = (
                 <LogSlider
-                    value={props.value * ((props.properties as any).scale || 1)}
+                    value={value * ((properties).scale || 1)}
```

Comment:
properties is a map, instead of casting to `any` and accessing properties via property accessors, we should be using `map.get()` to retrieve the values